config/routes.rb

```diff
@@ -86,6 +86,7 @@
         post :confirm_details
         get :exempt
         get :incorrect_details
+        post :confirm_incorrect_details
```

Comment:
if validation failed, path in browser will change what is not good
```suggestion
        post :incorrect_details, to: 'vehicles#confirm_incorrect_details'
```